apps/edxapp/templates/services/cms/job_01_create_directories.yml.j2

```diff
@@ -0,0 +1,47 @@
+{%- from "apps/edxapp/templates/services/cms/macros/image.yml.j2" import image_name -%}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: edxapp-cms-create-directories-{{ job_stamp }}
+  namespace: "{{ project_name }}"
+  labels:
+    app: edxapp
+    service: cms
+    version: "{{ edxapp_image_tag }}"
+    job_stamp: "{{ job_stamp }}"
+    # Jobs with the "pre" job type will be executed prior to deployments
+    job_type: "pre"
+    deployment_stamp: "{{ deployment_stamp }}"
+spec:
+  template:
+    metadata:
+      name: edxapp-cms-create-directories-{{ job_stamp }}
+      labels:
+        app: edxapp
+        service: cms
+        version: "{{ edxapp_image_tag }}"
+        job_stamp: "{{ job_stamp }}"
+        deployment_stamp: "{{ deployment_stamp }}"
+    spec:
+      containers:
+        - name: edxapp-cms-create-directories-{{ job_stamp }}
+          # We point to a local registry image build for this "live" image (see
+          # ImageStream and BuildConfig templates)
+          image: "{{ internal_docker_registry }}/{{ project_name }}/{{ image_name('cms', edxapp_image_tag, edxapp_theme_tag) }}"
+          command:
+            - "bash"
+            - "-c"
+            - {% for path in required_directories %}(mkdir {{ path }} || true) && {% endfor %}echo "✅"
+          volumeMounts:
+            - mountPath: /edx/var/edxapp/media
+              name: edxapp-v-media
+            - mountPath: /edx/app/edxapp/data
+              name: edxapp-v-data
+      volumes:
+        - name: edxapp-v-media
+          persistentVolumeClaim:
+            claimName: edxapp-pvc-media
+        - name: edxapp-v-data
+          persistentVolumeClaim:
+            claimName: edxapp-pvc-data
```

Comment:
I think we should mount all available volumes here?

Reply:
you're right I mounted what I needed but since required directories= is configurable we should mount all.